src/applications/vaos/containers/DateTimeSelectPage.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  openFormPage,
+  updateFormData,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+} from '../actions/newAppointment.js';
+import { getFormPageInfo } from '../utils/selectors';
+import slots from './../api/slots.json';
+import Calendar from './../components/calendar/CalendarWidget';
+import { focusElement } from 'platform/utilities/ui';
+import moment from 'moment-timezone';
+
+const pageKey = 'dateTimeSelect';
+
+const institutionTimezone = 'America/Denver';
```

Comment:
Temporarily hardcoded this timezone.  Will be grabbing this from selected institution/clinic.